engine/classes/ElggUser.php

```diff
@@ -183,6 +183,23 @@ protected function update() {
 	}
 
 	/**
+	 * Get user language or default to site language
+	 *
+	 * @param string $fallback Optional language
+	 *
+	 * @return string
+	 */
+	public function getLanguage($fallback = null) {
+		if (isset($this->language)) {
```

Comment:
Can you please check `!empty` instead of `isset` here? May not matter but depending on backend storage, could be an empty string.